packages/@lwc/engine-core/src/framework/hooks.ts

```diff
@@ -183,7 +183,6 @@ export function createViewModelHook(elm: HTMLElement, vnode: VCustomElement) {
     }
     const { sel, mode, ctor, owner } = vnode;
     const def = getComponentInternalDef(ctor);
-    setElementProto(elm, def);
```

Comment:
the only thing in between this proto mutation and createVM is the setElementShadowToken, which I don't think cares about the proto chain of the element itself.